wordle-web/app.vue

```diff
@@ -24,23 +50,62 @@
 <script setup lang="ts">
 import { useTheme } from "vuetify";
 import nuxtStorage from "nuxt-storage";
-
+import { provide, ref, watch, onMounted, inject } from "vue";
 const router = useRouter();
 const theme = useTheme();
 const showHelpDialog = ref(false);
+const showUserNameDialog = ref(false);
+const userName = ref("guest");
+
+provide("userNameValue", userName.value);
+provide("userName", userName);
+provide("showUserNameDialog", showUserNameDialog.value);
```

Comment:
These do not need to be done using `provide` in my opinion. From what I can see, userName just needs to be accessible from `index.vue` because that's where you set   pos the scores. One fix could be to just post your scores within `app.vue` as you can inject the game here like in other components. However, i don't know if that's any better.

Reply:
Good call, we'll see if we can get that working.

Reply:
I did this because I wanted access to the object to make changes not on the app.vue page, and I wanted the username present throughout the entire page not only on the game page